instance/models/openedx_appserver.py

```diff
@@ -533,3 +534,8 @@ def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
         if not self.pk:
             self.configuration_settings = self.create_configuration_settings()
         super().save(*args, **kwargs)
+        publish_data({
+            'type': 'openedx_appserver_update',
+            'appserver_id': self.pk,
+            'instance_id': self.owner.pk,
```

Comment:
Again it seems the instance id isn't used.

Reply:
@viadanna, the instance ID is used for this notification type in the JS code below. (see `instance/static/js/src/instance.js` line 236).